framework/src/main/kotlin/br/com/zup/beagle/widget/ui/PullToRefresh.kt

```diff
@@ -20,21 +20,22 @@ import br.com.zup.beagle.core.ServerDrivenComponent
 import br.com.zup.beagle.widget.Widget
 import br.com.zup.beagle.widget.action.Action
 import br.com.zup.beagle.widget.context.Bind
+import br.com.zup.beagle.widget.context.Context
 import br.com.zup.beagle.widget.context.ContextComponent
 import br.com.zup.beagle.widget.context.ContextData
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/da8016244c94ee0c0ba7e8b462f1973d884f7ca2/framework/src/main/kotlin/br/com/zup/beagle/widget/ui/PullToRefresh.kt#L25"></span>The import 'br.com.zup.beagle.widget.context.ContextData' is unused.</td>
    </tr>
  </tbody>
</table>